packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -100,6 +107,21 @@ const NominationBanner = (props) => {
                 status={activeNomination.status}
                 toggleEmailModalState={toggleEmailModalState}
               />
+              {activeNomination.driveFolderId &&
+                activeNomination.status != 'received' &&
+                activeNomination.status != 'Awaiting HIPAA' && (
+                  <span>
+                    <button
+                      onClick={() => {
+                        openWindow(`${activeNomination.driveFolderId}`);
```

Comment:
You don't need the backticks or the ${ stuff. You can just `openWindow(activeNomination.driveFolderId)`

Reply:
fixed